arch/sim/Kconfig

```diff
@@ -422,28 +422,92 @@ endchoice
 
 endif
 
+menuconfig SIM_UART
```

Comment:
can we change to menu? so we still use SIM_UART_NUMBER == 0 to disable uart simulation.

Reply:
Yes that is very easy. But I did it like that because it looked much cleaner and provided a good point to add the "help" entry.  

If I change it to a menu, where do you propose to move the help, so it is easily reachable by the users?

Reply:
Still put here like "Simulated Graphics/Input" at line 253.

Reply:
@xiaoxiang781216 I don't get it. "Simulated Graphics/Input" does not have a help entry.

Reply:
Oh, you mean help text? I thought that menu also can have help string, if not we can add to SIM_UART_NUMBER?

Reply:
Yes, it was about the help text. As I see, there cannot be a help entry on menus. And I this new option was a good place to have the help entry.

I did the change per your request, which removes the new option, but with a slightly more obscured help entry.

If you think this is OK, you may merge it. I re-tested it, and it is OK.